Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -161,6 +161,7 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_AVD_Audit_Enable_HostPool_BootDiagnostics") -or
                 ($control.ControlId -eq "Azure_RedisEnterprise_DP_Use_TLS_Encrypted_Connections") -or
                 ($control.ControlId -eq "Azure_DBforPostgreSQL_DP_Use_Secure_TLS_Version") -or
+                ($control.ControlId -eq "Azure_BackupVault_DP_Enable_Soft_Delete") -or
```

Comment:
Please make sure to add remediation zip before merging this and do not merge PR till control is released

Reply:
Added RemediationScripts zip file